Arrowgene.Ddon.Shared/Entity/PacketStructure/C2LGpCourseGetInfoReq.cs

```diff
@@ -0,0 +1,36 @@
+using Arrowgene.Buffers;
+using Arrowgene.Ddon.Shared.Entity.Structure;
+using Arrowgene.Ddon.Shared.Network;
+
+namespace Arrowgene.Ddon.Shared.Entity.PacketStructure
+{
+    public class C2LGpCourseGetInfoReq : IPacketStructure
+    {
+        public C2LGpCourseGetInfoReq()
+        {
+        }
+
+        public PacketId Id => PacketId.C2L_GP_COURSE_GET_INFO_REQ;
+
+        public class Serializer : PacketEntitySerializer<C2LGpCourseGetInfoReq>
+        {
+
+            public override void Write(IBuffer buffer, C2LGpCourseGetInfoReq obj)
+            {
+                // WriteEntity(buffer, obj.CharacterInfo);
```

Comment:
Does the request contain no info? If so please remove the commented dead code

Reply:
It does contain info, it just was used for the cash shop interface. I can remove it from this PR.

Reply:
Actually, this one was not a cashstop packet. I misread which one this was. Either way, I can remove it.

Reply:
If the packet is supposed to have data, add a TODO comment so we know its not just an empty packet. If it doesn't then remove it so its clear that this packet isnt supposed to have data.